docs/content/en/docs-v0.43.x/user-guide/configuration-reference.md

```diff
@@ -181,6 +181,7 @@ spec:
 | Field | Type | Description | Required |
 |-|-|-|-|
 | name | string | The event name. | Yes |
+| createPullRequest | bool | Whether to create a new branch or not when commit changes in event watcher. | No |
```

Comment:
We don't need to update this file. This feature will be released on (at the earliest) v0.44.1, so it should not appear on v0.43.x docs.